sdk/core/System.ServiceModel.Rest/src/NullableResultOfT.cs

```diff
@@ -9,35 +9,22 @@ namespace System.ServiceModel.Rest;
 /// TBD.
 /// </summary>
 /// <typeparam name="T"></typeparam>
-public class NullableResult<T>
+public abstract class NullableResult<T>
```

Comment:
I needed to make these abstract rather than concrete to resolve a generic type resolution issue with the `ErrorResult<T>` private type that came over in the HttpPipelineExtensions shared source file (now ProtocolPipelineExtensions).

Reply:
It might be worth considering if we can solve that differently if we wanted to make these non-abstract, but I kind of like the abstract pattern with a private subclass implementation that gets returned from methods, if there aren't serious drawbacks to that approach.